agent/util-scripts/test-bin/rpm

```diff
@@ -19,8 +19,13 @@ fi
 
 _rpm="${3}"
 if [[ "${_rpm}" == "pbench-sysstat" ]]; then
-    echo "12.0.3-1"
-    _rc=0
+    if [[ $PBENCH_UNIT_TEST_NAME == "test-61" ]]; then
+	echo "pbench-sysstat-12.0.3-1 is not installed"
```

Comment:
I realize that this PR is still marked "draft"...but, I'm not seeing this "not installed" message in the output from `test-61`...so I'm not sure that things are "working", here.

Reply:
I must have forgotten to add the difference. Sorry about that.